lib/date.nit

```diff
@@ -0,0 +1,215 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# The date and time behavior is splited in 2 classes:
+# * `Date` handles the year, month and day part of the date.
+# * `Time` handles the time in hour, minute and second.
+# 
+# These are extended by DateTime that merges the two concepts in one class.
+#
+#	###################
+#	# Some operations #
+#	###################
+#
+#	#DateTime:
+#	var date_1  = new DateTime(2014, 02, 09, 13, 50, 0)
+#	var date_2  = new DateTime(2014, 02, 09, 13, 51, 0)
+#	
+#	assert date_1 > date_2
+# 	
+#	#Date:
+#	var date_1 = new Date(2014, 01, 12)
```

Comment:
You will have a Nitunit error on that, because `date_1` is already defined in this nitunit at line 25.